kolibri/core/tasks/registry.py

```diff
@@ -243,6 +245,15 @@ def check_job_permissions(self, user, job, view):
             if not permission.has_permission(user, job, view):
                 raise PermissionDenied
 
+    def get_validated_enqueue_args(self, enqueue_args_data):
+        """
+        The `JobValidator` pops out `enqueue_args` during validation so
+        this method serves the purpose of returning validated `enqueue_args`.
+        """
+        enqueue_args_serializer = EnqueueArgsSerializer(data=enqueue_args_data)
+        enqueue_args_serializer.is_valid(raise_exception=True)
```

Comment:
This should already have been validated as part of a the parent serializer validation.